test/e2e/lib/components/page-preview-component.js

```diff
@@ -38,10 +38,8 @@ export default class PagePreviewComponent extends AsyncBaseContainer {
 
 	async close() {
 		await this.driver.switchTo().defaultContent();
-		return await driverHelper.clickWhenClickable(
-			this.driver,
-			By.css( 'button.web-preview__close' )
-		);
+		let closeButton = await this.driver.findElement( By.css( 'button.web-preview__close' ) );
+		return await this.driver.executeScript( 'arguments[0].click()', closeButton );
```

Comment:
For some reason the click wasn't working in some environments. Using a JS click works